content/en/blog/2024/otel-errors/index.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Dude, Where's My Error? How OpenTelemetry Records Errors
+linkTitle: OTel Errors
+date: 2024-03-06
+author: >- # If you have only one author, then add the single name on this line in quotes.
+  [Reese Lee](https://github.com/reese-lee) (New Relic),
+  [Adriana Villela](https://github.com/avillela) (ServiceNow)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+---
+Depending on the language(s) you’re used to developing in, you may have certain 
+ideas about what an error is, as well as what constitutes an exception and how 
+it should be handled. For example, Go does not have exceptions, partly to 
+discourage programmers from labeling too many ordinary errors as exceptional. 
+On the other hand, languages such as Java and Python provide built-in support 
+for throwing and catching exceptions. 
+
+When you begin at a place where different languages disagree about what an error 
+or exception is and how to handle them, where do you go when you need standardized 
+telemetry and error reporting across microservices written in those languages? 
+OpenTelemetry is the key to this answer, and is the lens through which we’ll 
+address the following, and more:
```

Comment:
These 2 sentences create 3 different pictures:
* Where do you go? (a place)
* OpenTelemetry is the key (unlock something)
* It is the lens (look through something)

Can you make this consistent somehow?

Reply:
I modified this to
>When you begin at a place where different languages disagree about what an error 
or exception is and how to handle them, what do you use when you need standardized 
telemetry and error reporting across microservices written in those languages? 
OpenTelemetry is the tool with which we'll address the following, and more:

Does that work?

Reply:
yes, thanks